.github/workflows/ci-e2e.yml

```diff
@@ -22,14 +22,14 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install gh-action-multiline
-        run: curl -fsSL https://raw.githubusercontent.com/kachick/gh-action-multiline/${{ github.sha }}/scripts/install-in-github-action.sh | sh -s
+        run: curl -fsSL https://raw.githubusercontent.com/kachick/gh-action-multiline/${{ github.sha }}/scripts/install-in-github-action.sh | sh -s v0.1.1
```

Comment:
https://github.com/kachick/wait-other-jobs/issues/474 should help 🤔

Reply:
Added in https://github.com/kachick/gh-action-escape/pull/70/commits/b63530627ed9dce0701cfcf9b5ba1c0821c37477